packages/deployments/contracts/contracts/core/connext/facets/TokenFacet.sol

```diff
@@ -383,6 +388,42 @@ contract TokenFacet is BaseConnextFacet {
     IBridgeToken(local).setDetails(_name, _symbol);
   }
 
+  /**
+   * @notice Used to update the owner of a representation asset
+   * @dev You *CANNOT* call this function if there is a nonzero supply of the local asset
+   * (indicating that some funds are still locked on the canonical domain, and you may
+   * lose your ability to burn / mint them)
+   * @param _canonical - The canonical id and domain to remove
+   * @param _newOwner - The new owner
+   */
+  function transferTokenOwnership(TokenId calldata _canonical, address _newOwner) external onlyOwnerOrAdmin {
```

Comment:
I guess what I'm getting at is if this `transferTokenOwnership` will eventually come from a DAO whose method calls would undergo proposal period then i think that's fine

Reply:
right so that would come up when we are trying to change ownership of a token that wasn't deployed by our bridge, but listed in it (i.e. we used `setupAssetWithDeployedRepresentation`.

as is, this is designed primarily for `local` assets that get deployed by our bridge which inherit the standard `Ownable` contract. the reason those tokens don't have the propose/accept structure is because the bridge is the ultimate owner

but yeah i thought about that too -- will add a note in the comments that this is really for basic ownership structures (i.e. you must really be careful with who you put as `_newOwner` since they don't have to accept it to prove its valid and there's no time lock)

Reply:
added a comment https://github.com/connext/nxtp/pull/2742/commits/269b3782ba99df16c32e7eab2f483c7163b472b1